runtime/lua/vim/ui.lua

```diff
@@ -140,11 +146,15 @@ function M.open(path)
   end
 
   local cmd --- @type string[]
-  local opts --- @type vim.SystemOpts
+  local conf --- @type vim.SystemOpts
 
-  opts = { text = true, detach = true }
+  conf = opts or {}
+  conf.text = opts.text or true
+  conf.detach = opts.detach or true
```

Comment:
looks like `opts` and `conf` are being mixed up. they aren't related.

Reply:
i guess `opts` could be a way to pass these options, but it wasn't really something we discussed, and it's not documented... should wait until a different PR so we can wrap this one up

Reply:
Ehm, can I ask for your suggestion here? I really don’t understand what would you like to be done.

Reply:
What is named `conf` here is unrelated to the `opts` passed to `vim.ui.open`. If we want to also allow callers to configure the `vim.system()` options, that would be a separate `opts` field such as `opts.cmd_opts` or something like that.

For now I removed that.